secrets/oidc-config.yaml.sample

```diff
@@ -0,0 +1,12 @@
+---
+## This file contains a sample OIDC configuration
+## Note: this is a sample file 
+oidc_attributes:
+    # [required] OIDC IssuerURL - token issuer URL
+    issuer_url: # Fill here
+    # [required] Service account client ID
+    client_id: # Fill here
+    # [required] Service account client Secret
+    client_secret: # Fill here
+    # [required] OIDC audience 
```

Comment:
cc @pb82 is audience required?
I seem to remember in some other tool we left it empty

Reply:
No, audience should be an optional field. Observatorium requires it, but the proxy is not tied to Observatorium.